README.md

```diff
@@ -0,0 +1,198 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] OPTIONAL: In the desktop view, when I add a new piece of furniture, there is no way to get back to the home page.

I highly suggest kindly, either adding a back button or making the menu visible.